### ISSUE TITLE
[BUG] configureGamePass errors with 404 (Not Found)

### ISSUE THREAD

**Issue by phoriah** [noblox/noblox.js#784]
**Describe the bug**
A simple `configureGamePass` request errors with 404 (Not Found) status

**To Reproduce**
```js
const noblox = require("noblox.js");

(async () => {
  await noblox.setCookie(""); // Add Cookie

  await noblox.configureGamePass(
    0, // Replace with Gamepass ID you have access to
    "Game Pass Title",
    "Game Pass Description",
    1234
  );
})();
```

**Expected behavior**
I expect it to update name, description and price of the gamepass under specified id.
This is probably due to the fact that Roblox overhauled their gamepass dashboard & API related to it.
**Screenshots**
```node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error: An unexpected error occurred with status code 404.
    at c:\DIR\node_modules\noblox.js\lib\games\configureGamePass.js:127:15
    at tryCatcher (c:\DIR\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (c:\DIR\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (c:\DIR\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (c:\DIR\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (c:\DIR\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (c:\DIR\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (c:\DIR\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (c:\DIR\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues [as _onImmediate] (c:\DIR\node_modules\bluebird\js\release\async.js:15:14)
    at process.processImmediate (node:internal/timers:478:21)```

**Comment by Neztore** [comment]
The API being used:
 https://github.com/noblox/noblox.js/blob/0d347116f880c45a96ca6aa22ac6928b9917ab1c/lib/games/configureGamePass.js#L91

Will have been removed as you suggest. Do you know what the new API is?

**Comment by Regalijan** [comment]
This was fixed in #813
